backend/src/zimfarm_backend/common/external.py

```diff
@@ -230,6 +228,18 @@ def advertise_book_to_cms(task: TaskFullSchema, file_name: str):
             )
 
     # record request result
+    create_or_update_task_file(
+        session,
+        task_id=task.id,
+        name=file_name,
+        status=file_data.status,
```

Comment:
Why setting again the status, I don't see what would have modified it.

Reply:
And errors when notifying CMS should be fatal to the background task (we should not try to notify other books), CMS is meant to accept all notifications, if one notification fails we want to stop sending notifications